src/server.ts

```diff
@@ -66,19 +62,18 @@ export class JsonRpcServer extends Server implements CustomTransportStrategy {
   }
 
   public async listen(callback: () => void) {
-    let app = express();
-
-    app.post(this.options.path, express.json(), async (req, res) => {
-      // let handlers = this.getHandlers();
+    let app = this.options.server ?? express();
+    app.use(bodyParser.json());
```

Comment:
This will activate a global JSON parser on all routes in the app, which is somewhat invasive.